pmd-julia/src/test/resources/net/sourceforge/pmd/lang/julia/cpd/testdata/mathExample.txt

```diff
@@ -0,0 +1,153 @@
+    [Image] or [Truncated image[            Bcol      Ecol
+L2
+    [function]                              1         8
+    [sphere_vol]                            10        19
+    [(]                                     20        20
+    [r]                                     21        21
+    [)]                                     22        22
+L5
+    [return]                                3         8
+    [4]                                     10        10
+    [/]                                     11        11
+    [3]                                     12        12
+    [*]                                     13        13
+    [pi]                                    14        15
+    [*]                                     16        16
+    [r]                                     17        17
+    [^]                                     18        18
+    [3]                                     19        19
+L6
+    [end]                                   1         3
+L9
+    [quadratic]                             1         9
+    [(]                                     10        10
+    [a]                                     11        11
+    [,]                                     12        12
+    [sqr_term]                              14        21
+    [,]                                     22        22
+    [b]                                     24        24
+    [)]                                     25        25
+    [=]                                     27        27
+    [(]                                     29        29
+    [-]                                     30        30
+    [b]                                     31        31
+    [+]                                     33        33
+    [sqr_term]                              35        42
+    [)]                                     43        43
+    [/]                                     45        45
+    [2]                                     47        47
+    [a]                                     48        48
+L12
+    [function]                              1         8
+    [quadratic2]                            10        19
+    [(]                                     20        20
+    [a]                                     21        21
+    [:]                                     22        22
+    [:]                                     23        23
```

Comment:
aren't these two a single token?

Reply:
Yes, they are. I fixed this issue. I checked for other mistakes. There was an error in detecting values as well. Also took care of that.